lib/spanner/bundle/config_validator.ex

```diff
@@ -51,6 +45,32 @@ defmodule Spanner.Bundle.ConfigValidator do
     validate_commands!(btype, commands)
   end
 
+  # Validate bundle Type
+  defp validate_bundle_type!(json, field) do
```

Comment:
Actually, thinking on it a bit more, we don't really need to do even this. Relay doesn't do anything with elixir bundles anymore anyway; the only thing that would handle an elixir bundle is the bot itself with the embedded bundle.

I think we can basically just ignore the type altogether.